Header header
string[] x_centers
string[] y_centers
string[] colors
